chart/templates/hpa-external-metric.yaml

```diff
@@ -1,9 +1,13 @@
 {{- $root := . -}}
 {{- range $name, $service := $root.Values.web.webServices }}
 {{- if $service.autoscaling.enabled }}
-{{- $pool := "sum:puma.pool_capacity{kube_cluster_name:dsva-vagov-" }}
-{{- $threads := "sum:puma.max_threads{kube_cluster_name:dsva-vagov-" }}
-{{- $deployment := "-cluster,kube_deployment:" }}
+{{- $pool := "avg:puma.pool_capacity{kube_cluster_name:dsva-vagov-" }}
+{{- $threads := "avg:puma.max_threads{kube_cluster_name:dsva-vagov-" }}
+{{- $replicas_available := "sum:kubernetes_state.deployment.replicas_available{kube_cluster_name:dsva-vagov-" }}
+{{- $pool_with_filters := $pool | cat $root.Values.target_env "-cluster,kube_deployment:" $name "}" }}
+{{- $threads_with_filters := $threads | cat $root.Values.target_env "-cluster,kube_deployment:" $name "}" }}
+{{- $replicas_with_filters := $replicas_available | cat $root.Values.target_env "-cluster,kube_deployment:" $name "}" }}
+
```

Comment:
non-blocking, but there's an extra line here

Reply:
I put that there to separate the regular vars from the calculations